src/Makefile

```diff
@@ -127,18 +129,20 @@ $(OUTPUT)/retsnoop.skel.h: $(OUTPUT)/mass_attach.bpf.o
 $(OUTPUT)/retsnoop.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 $(OUTPUT)/mass_attacher.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 
+ifeq ($(SIDECAR),$(DEFAULT_SIDECAR))
```

Comment:
I think you don't need this ifeq, just change `$(SIDECAR)::` into `$(DEFAULT_SIDECAR)::` and it seems to behave exactly how we want it

Reply:
Good point! I've also confirmed it works, updated the commit.